BranchSDK-Samples/Windows/ColorPicker/ColorPicker.cpp

```diff
@@ -226,6 +230,71 @@ INT_PTR CALLBACK About(HWND hDlg, UINT message, WPARAM wParam, LPARAM lParam)
     return (INT_PTR)FALSE;
 }
 
+
+#define BYTESIZE(str_len) (sizeof(TCHAR) * str_len)
+/**
+ * Set the registry up to allow for this app to be opened via. a web click
+ */
+void installApp()
+{
+    int bytesize = sizeof(TCHAR);
+
+    HKEY hKey;
+    LPCTSTR keyBase = TEXT("SOFTWARE\\Classes\\color");
+
+    // Step 1:  Create the Base Key and default values
+    LONG rc = RegCreateKeyEx(HKEY_CURRENT_USER, keyBase, 0, NULL, REG_OPTION_NON_VOLATILE, KEY_WRITE, NULL, &hKey, NULL);
+    if (rc != ERROR_SUCCESS) {
```

Comment:
So which is it, error or success? 😄(naming of ERROR_SUCCESS)

Reply:
That's a Windows definition.
Reference:  https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-